src/components/DpTimePicker/DpTimePicker.vue

```diff
@@ -25,6 +26,7 @@
       :value="currentTime" />
     <dp-input
       v-else
+      :data-cy="`${dataCy}:mobile`"
```

Comment:
since this is a if/else hook. should the name change for mobile devices?

Reply:
yes I thing we don't need it!
@spiess-demos

Reply:
right you are. as only one of them is rendered at a time.